app/models/user.rb

```diff
@@ -11,6 +11,19 @@ class User < ApplicationRecord
   validates :email, uniqueness: true
   validates_presence_of :first_name, :last_name
 
+  def self.create_with_omniauth(auth)
+
+    user = find_or_create_by(uid: auth['uid'], provider:  auth['provider'])
+    user.email = "#{auth['uid']}@#{auth['provider']}.com"
```

Comment:
lets leave the email out of it for now, don't want to confuse ourselves down the line thinking these are real emails.